README.md

```diff
@@ -0,0 +1,2 @@
+# Pedro-Santiago

```

Comment:
Please let you readme more descriptive

Reply:
- [ ] When you suggest a change make sure to be more clear and be more accurate 😉 🙃 
- [ ] Also use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.